index.js

```diff
@@ -1,8 +1,10 @@
 const healthPath = '/health';
```

Comment:
is it required?

Reply:
I don't think this will be required for now, was wondering if it might be a good way to check the health of our database in future via api. But seems pointless hence removing.